src/jon_mikkel_ex/ex05/walker_sim.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+
+__author__ = "Jon-Mikkel Korsvik"
+__email__ = "jonkors@nmbu.no"
+
+from random import getrandbits, seed as random_seed
+
+
+class Walker:
+    """
+    Walker class, with methods for checking position and moving.
+    Has its own method for finishing a trip home.
+    :param start_place: initial position of the walker
+    :param home: position of the walker's home
+    """
+    def __init__(self, start_place, home):
+        self.position = start_place
+        self.home = home
+        self.steps = 0
+
+    def move(self):
+        """
+        Uses a really efficient cointoss-  bool(getrandbits(1)) gives true or
+        false. Registers each move as a step.
+        """
+        if bool(getrandbits(1)):
+            self.position += 1
+        else:
+            self.position -= 1
+        self.steps += 1
```

Comment:
I recall commenting on this last week. I reccommend using `randint`or `choice` from the random package.